test/evp_extra_test.c

```diff
@@ -889,6 +889,10 @@ static int test_EVP_SM2_verify(void)
     if (!TEST_ptr(pctx = EVP_PKEY_CTX_new(pkey, NULL)))
         goto done;
 
+    /* FIXME: this will fail since pkey has no group data inside */
```

Comment:
Oh right, the key has explicit group parameters...  The easiest is probably to replace it with a public key that has the actual group name.

Reply:
...  or wait for #12604 to be merged, that will also solve the problem...

Reply:
#12604 sounds like a better idea to me

Reply:
Agreed

Reply:
This needs some adjustments in #12675 as well

Reply:
I'm going to kill #12675, it's not appropriate. There will be a fix for it at some point, to make it return with an error on provider-native EVP_PKEYs.